pkg/client/restclient/config.go

```diff
@@ -45,11 +45,11 @@ const (
 // Config holds the common attributes that can be passed to a Kubernetes client on
 // initialization.
 type Config struct {
-	// Host must be a host string, a host:port pair, or a URL to the base of the apiserver.
-	// If a URL is given then the (optional) Path of that URL represents a prefix that must
-	// be appended to all request URIs used to access the apiserver. This allows a frontend
-	// proxy to easily relocate all of the apiserver endpoints.
-	Host string
+	// Hosts must be a slice of host strings, host:port pairs, or a URLs to the base of the
+	// apiservers. If a URL is given then the (optional) Path of that URL represents a prefix
+	// that must be appended to all request URIs used to access the apiserver. This allows a
+	// frontend proxy to easily relocate all of the apiserver endpoints.
+	Hosts []string
```

Comment:
to summarize:
1. in this change --master and --api-servers will overwrite values provided in kubeconfig Server/Servers fields, not clear if i should add --master flag to kubelet
2. Server will be passed as Host, Servers as AlternateHosts
3. i will add Host back to restclient.Config which, if not empty and not present in AlternateHosts, will be used as a first url passed to url provider, this indeed will reduce amount of changes, especially in tests